.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+## Context
+<!-- Please brifly describe the problem the PR solves and what the solution was -->
```

Comment:
I assume this is a hint for the PR editor. But will GitHub hide the content in \<!-- ... --> ?  A `\` may be needed.

Reply:
The help will show when the author edits the message, but not in "show" mode

Reply:
ok, that's good then.